src/Messages/Attachments/OWSAttachmentsProcessor.m

```diff
@@ -118,6 +118,11 @@ - (void)retrieveAttachment:(TSAttachmentPointer *)attachment
         }
     };
 
+    if (attachment.serverId < 100) {
+        DDLogError(
```

Comment:
I've modified the attachment download logic to be more verbose around the symptom.